app/Resources/views/base.html.twig

```diff
@@ -35,7 +35,7 @@
                     <div class="container">
                         <div class="navbar-header">
                             <a class="navbar-brand" href="{{ path('homepage') }}">
-                                Symfony Demo
+                                Symfony Demo <small style="font-size: 10px;"><strong>ESI TIME:</strong> {{ "now"|date('H:i:s') }}</small>
```

Comment:
@sergiu-popa,  I think, you could add this dates wrapped by html comments, like this:
```twig
<!-- ESI TIME: {{ "now"|date('H:i:s') }} -->
```